src/store/store.ts

```diff
@@ -566,44 +566,44 @@ export const useStore = defineStore('store', {
      * - certify
      * - staff payment
      */
-    // hasCorrectionDataChanged (): boolean {
-    //   if (this.isBaseCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasBusinessTypeChanged ||
-    //       this.haveNameTranslationsChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged ||
-    //       this.hasShareStructureChanged ||
-    //       this.areProvisionsRemoved ||
-    //       this.haveNewResolutionDates
-    //     )
-    //   }
-
-    //   if (this.isFirmCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasBusinessStartDateChanged ||
-    //       this.hasNaicsChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged
-    //     )
-    //   }
-
-    //   if (this.isCoopCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasAssociationTypeChanged ||
-    //       this.hasSpecialResolutionMemorandumChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged ||
-    //       this.hasSpecialResolutionRulesChanged ||
-    //       this.hasSpecialResolutionResolutionChanged
-    //     )
-    //   }
-
-    //   return false // should never happen
-    // },
+    hasCorrectionDataChanged (): boolean {
+      if (this.isBaseCorrectionFiling) {
+        return (
+          this.hasBusinessNameChanged ||
+          this.hasBusinessTypeChanged ||
+          this.haveNameTranslationsChanged ||
+          this.haveOfficeAddressesChanged ||
+          this.havePeopleAndRolesChanged ||
+          this.hasShareStructureChanged ||
+          this.areProvisionsRemoved ||
+          this.haveNewResolutionDates
+        )
+      }
+
+      if (this.isFirmCorrectionFiling) {
+        return (
+          this.hasBusinessNameChanged ||
+          this.hasBusinessStartDateChanged ||
+          this.hasNaicsChanged ||
+          this.haveOfficeAddressesChanged ||
+          this.havePeopleAndRolesChanged
+        )
+      }
+
+      if (this.isCoopCorrectionFiling) {
+        return (
+          this.hasBusinessNameChanged ||
+          this.hasAssociationTypeChanged ||
+          this.hasSpecialResolutionMemorandumChanged ||
+          this.haveOfficeAddressesChanged ||
+          this.havePeopleAndRolesChanged ||
+          this.hasSpecialResolutionRulesChanged ||
+          this.hasSpecialResolutionResolutionChanged
+        )
+      }
+
+      return false // should never happen
+    },
```

Comment:
Haha, good thing you left the old code in there.

Of course, if you had deleted it then you could have seen the old code in your original PR.

Reply:
Oh, please delete comment on line 561.

Reply:
Fixed. Resolving.